src/main/java/org/elasticsearch/action/support/nodes/TransportNodesOperationAction.java

```diff
@@ -203,7 +203,7 @@ private void onFailure(int idx, String nodeId, Throwable t) {
                 logger.debug("failed to execute on node [{}]", t, nodeId);
             }
             if (accumulateExceptions()) {
-                responses.set(idx, new FailedNodeException(nodeId, "Failed node [" + nodeId + "]", t));
+                responses.set(idx, new FailedNodeException(nodeId, "Failed node [" + nodeId + "][" + t.getMessage() + "]", t));
```

Comment:
I Am concerned that we will miss the actual message because its wrapped, my vote is the detailed message one